user/urls.py

```diff
@@ -1 +1,11 @@
-# write your code here
+from django.urls import path
+
+from user.views import CreateUserView, LoginUserView, ManageUserView
+
```

Comment:
Add 1 blank line here. Between import statements and the rest of the code - 2 blank lines.